### ISSUE TITLE
Can not reproduce any content - 'One or more items failed to play' error

### ISSUE THREAD

**Issue by Vichman67** [Sandmann79/xbmc#764]
**Addon used**
- [X] Amazon VOD (plugin.video.amazon-test)
- [ ] Browser Launcher (plugin.program.browser.launcher)

- Addon version: 1.1.2+matrix.1

**Account type**
- [X] primevideo.com
- [ ] amazon.(com/co.uk/de/jp)

**System Setup (please provide the following information):**
- Hardware: Raspberry Pi 5
- OS version: LibreELEC 11.0.6
- Kodi version number: Kodi (Nexus) v20.3

**Upload Logs**

- Relevant Kodi log: https://pastebin.com/LbQB0z0P
- AmazonVOD plugin log directory zipped: https://www.mediafire.com/file/4atzk8yz8u7up5e/AmazonVod_log_dir.zip/file

**Describe the bug**

It's been 4 weeks since this error appeared on my system, and I've not seen any similar bug so far, so here I go...

Starting the addon and browsing the catalog (included my space items - movies or TV shows) works fine. But when trying to play any item, a pop-up window appears with the following message:
```
Playback failed
One or more items failed to play. Check the log for more information about this message.
```

Pressing the pop-up window OK button returns to the previous catalog browsing screen. So I'm not able to play Amazon contents.

Reinstalling the addon didn't help.

This error doesn't happen on Kodi 21.0-ALPHA1 (Omega) under Laptop running Ubuntu 22.04.5.

Reviewing relevant Kodi log I've seen several messages like this one right after the items play call. This is the item play call:
```
2024-10-23 19:28:00.568 T:1109     info <general>: [Amazon VOD] network.py:119 getURL: https://www.primevideo.com/mystuff/watchlist/ref=atv_hm_hom_c_9zZ8D2_mys_wl?episodeListSize=9999
```
And this is the error message right after the play call:
```
2024-10-23 19:28:00.623 T:1109    error <general>: /storage/.kodi/addons/script.module.urllib3/lib/urllib3/connectionpool.py:1099: InsecureRequestWarning: Unverified HTTPS request is being made to host 'www.primevideo.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
```
There are also errors related to WideVine:
```
2024-10-23 19:28:08.411 T:1124    error <general>: AddOnLog: inputstream.adaptive: Initialize: Failed to load library: /storage/.kodi/cdm/libwidevinecdm.so: wrong ELF class: ELFCLASS64
2024-10-23 19:28:08.411 T:1124    error <general>: AddOnLog: inputstream.adaptive: Unable to load widevine shared library (/storage/.kodi/cdm/libwidevinecdm.so)
2024-10-23 19:28:08.411 T:1124    error <general>: AddOnLog: inputstream.adaptive: OpenDRMSystem failed
2024-10-23 19:28:08.411 T:1124    debug <general>: AddOnLog: inputstream.adaptive: CSession::~CSession()
2024-10-23 19:28:08.412 T:1124    error <general>: CVideoPlayer::OpenInputStream - error opening [plugin://plugin.video.amazon-test/?asin=amzn1.dv.gti.3fb3b886-cffd-49e9-a9ed-a176614fa6c6&mode=PlayVideo&name=Sof%C3%A1+tres+plazas&adult=0&trailer=0&selbitrate=0]
```

And about AmazonVOD log files, I'm not able to conclude anything after taking a look at them.

Any help will be much appreciated.

Víctor.

**Comment by Vichman67** [comment]
Problem solved!

As well as completely deinstalling AmazonVOD addon, I proceded to delete WideVine from my Raspberry Pi (from file manager -> delete folder /storage/.kodi/cdm) and to restart the Raspberry.

Then I ran AmazonVOD addon config and entered my Amazon Prime Video credentials. No more settings adjustments made.

Et voilà: as usual, after searching on my stuff for any content and trying to reproduce it, the addon suggested me to install WideVine. I answered 'yes' and after WideVine successful installation, the content got properly played.

So it seemed It all was about some kind of WideVine corruption...

Hope this will be useful for anybody facing the same problem, or even for expanding the FAQ section of this project.

Thanks!
